lib/utils.ts

```diff
@@ -372,8 +377,17 @@ export async function getBody(
 	);
 }
 
-// This is the actual implementation that hides the internal `retriesRemaining` parameter
+const isFile = (value: string | WebResourceFile): value is WebResourceFile => {
+	return (
+		value != null &&
+		typeof value !== 'string' &&
+		value.name != null &&
+		typeof value.name === 'string' &&
+		value instanceof Blob
```

Comment:
I think it'd make sense to move this check earlier on and could probably replace the null/string checks?

Reply:
Yes, much better. Thanks.